build-resources.gradle

```diff
@@ -7,7 +7,7 @@
 ext.versionMap = [
         junitJupiter : '5.8.2',
         mockito : '3.11.2',
-        opentelemetryProto : '1.7.1-alpha',
+        opentelemetryProto : '0.16.0-alpha',
```

Comment:
Did you have to make any code changes to support this? I tried making this change a few weeks ago. But, some of the tests were failing with an exception and I didn't get to look further into it.